docs/articles/documentation/test-api/actions/take-screenshot.md

```diff
@@ -6,23 +6,99 @@ checked: true
+This topic describes how you can take screenshots of the tested page.
 
-**Note**: this action requires a [ICCCM/EWMH-compliant window manager](https://en.wikipedia.org/wiki/Comparison_of_X_window_managers) on Linux.
+**Note**: these actions require a [ICCCM/EWMH-compliant window manager](https://en.wikipedia.org/wiki/Comparison_of_X_window_managers) on Linux.
+
+> Important! If the screenshot directory is not specified with the [runner.screenshots](../../using-testcafe/programming-interface/runner.md#screenshots) API method or the [screenshots](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) command line option,
+> the screenshot actions are ignored.
+
+## Take a Screenshot of the Entire Page
 
 ```text
 t.takeScreenshot( [path] )
 ```
 
-Parameter           | Type   | Description                                                                                           | Default
-------------------- | ------ | ----------------------------------------------------------------------------------------------------- | ----------
-`path`&#160;*(optional)* | String | The relative path and the name of the screenshot file to be created. Resolved from the *screenshot directory* specified by using the [runner.screenshots](../../using-testcafe/programming-interface/runner.md#screenshots) API method or the [screenshots-path](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) command line option. | `{currentDate}\test-{testIndex}\{userAgent}\{screenshotIndex}.png`; `{currentDate}\test-{testIndex}\run-{quarantineAttempt}\{userAgent}\{screenshotIndex}.png` if quarantine mode is enabled.
+Parameter           | Type   | Description
+------------------- | ------ | -----------------------------------------------------------------------------------------------------
+`path`&#160;*(optional)* | String | The relative path and the name of the screenshot file to be created. Resolved from the *screenshot directory* specified by using the [runner.screenshots](../../using-testcafe/programming-interface/runner.md#screenshots) API method or the [screenshots](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) command line option.
+
+By default, paths to which screenshots are saved are specified as:
 
-> Important! If the screenshot directory is not specified with the [runner.screenshots](../../using-testcafe/programming-interface/runner.md#screenshots) API method or the [screenshots-path](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) command line option,
-> the `t.takeScreenshot` action is ignored.
+* `{currentDate}\test-{testIndex}\{userAgent}\{screenshotIndex}.png` if the [quarantine mode](../../using-testcafe/command-line-interface.md#-q---quarantine-mode) is disabled;
+* `{currentDate}\test-{testIndex}\run-{quarantineAttempt}\{userAgent}\{screenshotIndex}.png` if the [quarantine mode](../../using-testcafe/command-line-interface.md#-q---quarantine-mode) is enabled.
```

Comment:
Maybe we need subdirectory for elements screenshots?
Smth like this `{currentDate}\test-{testIndex}\{userAgent}\elements\{screenshotIndex}.png`

Reply:
\cc @AlexanderMoskovkin we are waiting for you here

Reply:
Discussed in private. Let's don't add a subdirectory